[DOCS] Corrects broken import for prerequisites in v0.18 connect to Filesystem Data Assets page (#9679)
great-expectations/great_expectations: docs/docusaurus/versioned_docs/version-0.18/oss/guides/connecting_to_your_data/fluent/filesystem/connect_filesystem_source_data.md

<Prerequisites>

> - [An installation of GX set up to work with Azure Blob Storage](/oss/guides/setup/installation/install_gx.md)

Reviewer: ```suggestion
- [GX installed and set up to work with Azure Blob Storage](/oss/guides/setup/installation/install_gx.md)
```

<Prerequisites>

> - [An installation of GX set up to work with Azure Blob Storage](/oss/guides/setup/installation/install_gx.md)

Reviewer: ```suggestion
- [GX installed and set up to work with Azure Blob Storage](/oss/guides/setup/installation/install_gx.md)
```
